eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -95,4 +95,8 @@
     <UsagePattern IdentityGlob="System.Threading.Tasks/*" />
     <UsagePattern IdentityGlob="System.Threading.Tasks.Dataflow/*" />
   </IgnorePatterns>
+  <Usages>
+    <Usage Id="Microsoft.Bcl.AsyncInterfaces" Version="7.0.0" />
+    <Usage Id="System.Threading.Tasks.Extensions" Version="4.5.4" />
+  </Usages>
```

Comment:
@NikolaMilosavljevic @MichaelSimons

I copied the automatically generated baseline, like the build script told me to when run locally.  Should I change it to use the `UsagePattern` items like all the above, given these are both .NET owned packages?

Reply:
This worked in WSL, but is failing on CI? 😭

Reply:
These should be added in the section above using the wildcard syntax.

```
    <UsagePattern IdentityGlob="Microsoft.Bcl.AsyncInterfaces/*" />
    <UsagePattern IdentityGlob="System.Threading.Tasks.Extensions/*" />
```